assets/css/styles.css

```diff
@@ -0,0 +1,299 @@
+/* color palette */
+:root {
+  --powder-blue: #b8d8d8ff;
+  --cadet-blue: #c8c8c8;
+  --deep-space-sparkle: #4f6367ff;
+  --beige: #eef5dbff;
+  --orange-red-crayola: #fe5f55ff;
+}
+
+* {
+  font-family: "Work Sans", sans-serif;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0px;
+  background-color: var(--cadet-blue);
+}
+
+/* responsive navbar*/
+.navbar-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  background-color: var(--deep-space-sparkle);
+  /* sticky navbar */
+  /* position: fixed;
+  top: 0px;
+  z-index: 10; */
+}
+
+#name {
+  color: var(--cadet-blue);
+  font-size: 24px;
+  position: relative;
+  flex-grow: 2;
+  margin-left: 8px;
+}
+
+.nav-link-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: right;
+  flex-wrap: wrap;
+}
+
+.nav-links {
+  margin: 8px;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: var(--beige);
+  font-size: 18px;
+  font-weight: 400;
+}
+
+.nav-link:hover,
+#cv:hover {
+  color: var(--orange-red-crayola);
+  cursor: pointer;
+}
+
+/* hero image */
+#hero {
+  height: 100vh;
+  width: 100vw;
+  background-image: url("../images/Kinkakuji.jpg");
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+  /* margin-top: 18%; */
+}
+
+#herocaption {
+  color: var(--beige);
+  position: relative;
+  top: 93%;
+  right: 24px;
+  text-align: right;
+  font-size: 12px;
+}
+
+/* left-hand section titles */
+.section-title {
+  color: var(--beige);
+  font-size: 18px;
+  font-weight: 400;
+  text-align: right;
+  padding: 4px 12px;
+  width: 18%;
+  border-right: 8px solid var(--deep-space-sparkle);
+}
+
+/* responsive section containers */
+.aboutme-container,
+.portfolio-container,
+.contact-container {
+  display: flex;
+  flex-direction: row;
+  margin: 18px;
+}
+
+.aboutme-flex-container {
```

Comment:
This block also shares the same styling as the block below `portfolio-flex-container` - we need to group these together too, I would suggest renaming the class name to something more generic and applying it to both elements